twisted/plugins/fastpoetry_plugin.py

```diff
@@ -53,9 +53,12 @@ class Options(usage.Options):
 # Now we define our 'service maker', an object which knows
 # how to construct our service.
 
+@implementer(service.IServiceMaker, IPlugin)
 class PoetryServiceMaker(object):
 
-    implements(service.IServiceMaker, IPlugin)
+    # implements does not work in Python3
+    # see the implementer decorator above
+    ##implements(service.IServiceMaker, IPlugin)
```

Comment:
Let's just delete these lines then, no need to leave them here.

Reply:
Done.